README.md

```diff
@@ -20,28 +20,56 @@ then go to the generated folder
 
 and install the dependencies
 
-`yarn` or `npm install`
+`yarn install` or `npm install`
+
+then build the project
+
+`yarn build` or `npm run build`
+
+**You can found in your `package.json` some scripts to help you to develop your customizations.
+You can replace them with your own scripts if you want by using directly the `forest-cloud` command.**
 
 # Write your customizations
 
-Read the developer guide: https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization
+**[Read the developer guide](https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization)** to learn how to write your customizations.
+
+# Publish your customizations
+
+To publish your customizations, you need to build, package and publish your customizations.
+The build command will compile your customizations, the package command will create a package and the publish command will publish your package to your project.
 
-# Push your customizations to your project
+```bash
+forest-cloud build
```

Comment:
Should we indicate the version with `npx @forestadmin/cloud-toolkit`  here? Chances are that the package won't be globally installed.

Reply:
In the previous section I ask to the user to install the dependencies, it means we can use the version locally by adding npx in front of each commands